pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -646,6 +647,17 @@ func (r *ReconcileDependencyBuild) handleStateBuilding(ctx context.Context, db *
 			},
 		},
 	}
+	// TODO: ### Enclose this within an annotation to denote test CI system in use?
```

Comment:
Really we only need to limit the task resources on CI due to the fact that minikube and the e2e can't allocate the CPU/memory that the buildah-oci-ta task specifies by default. I think I could create another annotation to represent the CI tests being used in order to wrap this block...

Reply:
Latest commit has wrapped them in a new annotation.